internal/resources/fetcher.go

```diff
@@ -88,8 +132,14 @@ func (f *Fetcher) getResourcesDynamically(ctx context.Context,
 		Version:  version,
 		Resource: resource,
 	}
-	list, err := f.dynamicClient.Resource(resourceID).Namespace(namespace).
-		List(ctx, metav1.ListOptions{})
+	var list *unstructured.UnstructuredList
+	var err error
+	// if no namespace, we need to look for cluster wide resources
+	if len(namespace) != 0 {
```

Comment:
I would  prefer `getResourcesDynamically(ns) and `getClusterwideResourcesDynamically()`.